metapool/sample_sheet.py

```diff
@@ -647,10 +666,18 @@ def make_sample_sheet(metadata, table, sequencer, lanes, strict=True):
     messages = _validate_sample_sheet_metadata(metadata)
 
     if len(messages) == 0:
-        sheet = KLSampleSheet()
+        if metadata['Assay'] == 'TruSeq HT':
+            sheet = AmpliconSampleSheet()
+        elif metadata['Assay'] == 'Metagenomic':
+            sheet = AmpliconSampleSheet()
+        elif metadata['Assay'] == 'Metagenomic':
+            sheet = AmpliconSampleSheet()
+        else:
```

Comment:
Something seems weird here: It looks like the elifs are identical (both `elif metadata['Assay'] == 'Metagenomic':`) and I am lost as to why metagenomic assays would also get the amplicon sample sheet ...

Reply:
Thanks for the catch. :) They should all create corresponding sample-sheets.